cmd/devtool/devtool.go

```diff
@@ -412,3 +414,74 @@ func remoteConsole(destAccountAddr string) error {
 
 	return err
 }
+
+func copyDir(src, dst string) error {
```

Comment:
This function removes source dir at the end, so probably better be called `moveDir`?